### DISCUSSION TITLE
NTFY lxc?

### DISCUSSION THREAD

**Discussion by spants** [tteck/Proxmox#2294] [Ideas]
Has this been proposed? https://ntfy.sh

ntfy (pronounced notify) is a simple HTTP-based [pub-sub](https://en.wikipedia.org/wiki/Publish%E2%80%93subscribe_pattern) notification service. It allows you to send notifications to your phone or desktop via scripts from any computer, and/or using a REST API. It's infinitely flexible, and [100% free software](https://ntfy.sh/#free-software).

I created an Alpine LXC and installed the app - it runs well. Would be a nice template!

**Comment by tteck** [comment]
Similar to Gotify.

**Reply by JeGr** [reply]
While that is right, Gotify is no longer supported as a member/backend of Unified Push. Various Apps now add Unified Push as to not have to support every single app etc.

Ntfy is a unified push compatible backend for notification while the Gotify integration was deprecated (https://github.com/UnifiedPush/gotify-android)

So seeing many OSS android apps dipping their toe into de-googled notifications via UnifiedPush, I'd say it recons the new LXC :)

**Reply by tteck** [reply]
The reason there's a Gotify LXC and no Ntfy LXC is that Proxmox 8 has a Gotify notification target. 

![Screenshot 2024-01-15 10 19 12 AM](https://github.com/tteck/Proxmox/assets/21342670/3b328833-892f-4f5e-a594-b56a56d793b8)

**Reply by JeGr** [reply]
Indeed. And the follow up request for Proxmox to support ntfy was entered in their bugtracker already for the exact same reasons in December :) 
https://forum.proxmox.com/threads/notifications-add-ntfy-sh.137398/
https://bugzilla.proxmox.com/show_bug.cgi?id=5098

I just wanted to add input/perspective and point out that Gotify at this point in time seems(!) like it'll be superseeded by ntfy / unifiedpush very soon as they stoppped supporting newer UP versions and app developers everywhere (be it mobile, desktop, server) aren't keen on supporting multiple isolated solutions if there is a more generic standard. Which IMHO unifiedpush looks very much like. I won't say that can't change but right now I hear much talk and pushes to use open standards instead of single applications.

If that was unwanted then feel free to discard this at will, but I thought that's what Discussions are exactly there for :)

Cheers

**Reply by tteck** [reply]
I appreciate open discussions, and your input is valuable.

**Comment by dmshimself** [comment]
But ntfy has an IOS client which gotify still does not

**Comment by tteck** [comment]
ntfy LXC ready for testing

```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/ntfy.sh)"
```

**Reply by towerhand** [reply]
The script worked great on my end. I'll try configuring and playing with ntfy in the coming weeks. Thanks!!!

**Reply by DAE51D** [reply]
@tteck you are a god among men. We don't deserve you 👍🏼 

I'll add there is a great video by Network Chuck here: https://www.youtube.com/watch?v=poDIT2ruQ9M to now how to actually USE `ntfy` once you install this. And his video was the impetus to install this on my proxmox, but I thought, "I wonder if there is one of those sweet sweet bash script installers for this already..." and boom, here you are! 💟
